src/Symfony/Bundle/FrameworkBundle/DependencyInjection/FrameworkExtension.php

```diff
@@ -39,6 +39,11 @@ class FrameworkExtension extends Extension
     private $sessionConfigEnabled = false;
 
     /**
+     * @var string|null
+     */
+    private $kernelRootHash;
```

Comment:
It is mainly useful on shared server when several Symfony apps use the cache (APC/APCu basically). It avoids conflict.

I update the PR to rename it `kernelId`.